proto/lighthouse-response.proto

```diff
@@ -0,0 +1,363 @@
+syntax = "proto3";
```

Comment:
so we're not going with proto2 afterall?

Reply:
That's still up for debate, @paulirish @brendankenny, I used proto3 so I didn't have to include `optional`s and `required`s.  But brendan was talking about how there might be problems with default values, but I don't think we can come up with a good example of the problems at this time.

Reply:
I think with the latest changes I have resolved the issues with proto3's and default values.  So I think we might be g2g on proto3.